ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils_filtered_tenants.py

```diff
@@ -126,6 +126,10 @@ def _filtered_vrfs(self, tenant: dict) -> list[dict]:
                     and l3_interface.get("interfaces") is not None
                 )
             ]
+
+            vrf["_l3_multicast_enabled"] = default(get(vrf, "l3_multicast.enabled"), get(tenant, "l3_multicast.enabled"), None)
```

Comment:
No need to default to None. That is already the case.

Reply:
Also are we checking if underlay multicast is enabled anywhere? This would not work without underlay_multicast.

Reply:
I've now added tests for evpn_multicast, which is in turn testing for underlay_multicast and igmp_snooping.